.travis.yml

```diff
@@ -98,7 +98,7 @@ jobs:
         - echo "JS unit tests complete"
 
     - name: PHP unit tests (7.4, WordPress latest, with code coverage)
-      php: "7.4"
+      php: "7.1"
```

Comment:
Is there a specific reason we're downgrading this to 7.1? The name in the line above still references 7.4.

Reply:
@kaspars - Travis uses php-build and they updated Xdebug to 3.0.0 for PHP 7.2 and up. So this will allow the coverage tests to complete. (I missed the 7.2 in the name, which I updated.)
see https://github.com/php-build/php-build/blob/master/share/php-build/definitions/7.1snapshot

Reply:
Looks like we already have a PHP unit case for 7.1 here https://github.com/xwp/wp-foo-bar/blob/bcf360904b9e8dfeb5a57b6e3323a80659469a34/.travis.yml#L118-L120 so we're loosing 7.4 test.

Could we instead add a `before_install` step which rolls back xdebug to <3.0.0 when running these set of tests using the same `pecl install xdebug-2.9.8` script? Would that work on Travis?

Reply:
Oh, this particular task is just for generating the coverage report in addition to running the unit tests. The idea should still work, though.

Reply:
No PECL won't install a lower version if a higher version is already present. I tried that.

Reply:
Unless we use pecl to remove it first, then install a new one 🤔

Reply:
Nope, that didn't work. Pecl changes the php.ini wrong and breaks PHP 😞